src/test/java/org/sagebionetworks/bridge/exporter/integration/Exporter3Test.java

```diff
@@ -612,7 +745,23 @@ public void uploadWithScheduleContext() throws Exception {
         schedule = schedulesApi.saveScheduleForStudy(STUDY_ID, schedule).execute().body();
         
         Timeline timeline = schedulesApi.getTimelineForStudy(STUDY_ID).execute().body();
-        
+
+        // Update assessment config with basic survey config.
+        AssessmentConfig assessmentConfig = assessmentsApi.getAssessmentConfig(assessment.getGuid()).execute().body();
+        assessmentConfig.setConfig(ASSESSMENT_CONFIG);
+        assessmentsApi.updateAssessmentConfig(assessment.getGuid(), assessmentConfig).execute();
+
+        // Get study.
+        Study study = studiesApi.getStudy(STUDY_ID).execute().body();
+
+        // Make upload. This is a zip file with two files in it, metadata.json and assessmentResult.json.
+        byte[] zipFileContent = zip(SURVEY_UPLOAD_CONTENT_BY_FILENAME);
+
+        // todo the zip file seems to be malformed.
```

Comment:
Is this todo still relevant?

Reply:
Good catch. I left the TODO at the end of one coding session to remind myself of where I left off. Then for the next coding session, I immediately forgot about the TODO, but then went and solved the problem anyway. I guess the TODO didn't do it's job.